contrib/cirrus/runner.sh

```diff
@@ -59,6 +59,10 @@ _run_integration() {
     make integration
 }
 
+_run_podman_e2e() {
+	NETWORK_BACKEND=netavark make -C /var/tmp/go/src/github.com/containers/podman localintegration
```

Comment:
There's a clash over the meaning of `$DEBUG`, the automation libraries use 0 | 1 value but the e2e tests use empty/non-empty:  https://github.com/containers/podman/blob/main/test/e2e/common_test.go#L852  The automation libraries default it's value to '0', that's one reason why you see different behavior using `make localintegration` (by hand) vs `runner.sh` - the later is loading the automation libraries.  So...I think the above line should be:

```
export NETWORK_BACKEND=netavark
export -n DEBUG
make -C /var/tmp/go/src/github.com/containers/podman localintegration
```

Reply:
This is my annual reminder that noone should ever use `DEBUG` without a namespace prefix.

Reply:
Lol...hang on...let me clean off my hindsight spectacles :eyeglasses: Oh! Damnit! :angry: